charts/eol-exporter/values.yaml

```diff
@@ -0,0 +1,230 @@
+---
+eol-exporter:
+  enabled: true
+  namespaceOverride: ''
+  applicationName: eol-exporter
+  externalSecrets:
+    enabled: false
+  deployment:
```

Comment:
Doesn't it makes more sense to [push](https://prometheus.io/docs/instrumenting/pushing/) metrics to prmetheus (ex. run a cronjob once a day) rather than keep deployment running permanently (and perform healtchecks)?

Reply:
I think metrics being "stalled" is the reason.

Reply:
https://prometheus.io/docs/prometheus/latest/querying/basics/#staleness

Prom has 5m (by default) before a metrics is defined a 'stale'.